### PR TITLE
[doc] Document better ocamlfind and flambda requirements.

### LINKED ISSUE #6782: ocamlfind: Package `camlp5.gramlib' not found
#### Description of the problem

As a developper of Camlp5, I had to update it to make it compatible with ocaml 4.06. I did it, locally. I had to compile and install Num and findlib. Now, compiling Coq trunk, I get, after some compilation the message:

ocamlfind: Package `camlp5.gramlib' not found

What is that?

### PR DESCRIPTION
**Pull request by ejgallego**
Closes #6782

### CODE REVIEW

**Comment by maximedenes** [comment]
@ejgallego PRs should be self-assigned (not by the PR author), and there is no reason why this one would be for me when the only code owner is @Zimmi48. Please re-read `dev/doc/MERGING` to understand how assignment works.

File: INSTALL [hunk 1]
```diff
@@ -45,13 +45,21 @@ WHAT DO YOU NEED ?
    - for Coqide, the Lablgtk development files, and the GTK libraries
      including gtksourceview, see INSTALL.ide for more details
 
+   Note that camlp5 and lablgtk should be properly registered with
+   finlib/ocamlfind as the Coq makefile will use it to locate the
```
Comment (herbelin) [inline INSTALL]: findlib

Comment (ejgallego) [reply INSTALL]: Merci !

File: INSTALL [hunk 2]
```diff
@@ -45,13 +45,21 @@ WHAT DO YOU NEED ?
...
    Opam (https://opam.ocaml.org/) is recommended to install ocaml and
    the corresponding packages.
 
    $ opam install ocamlfind camlp5 lablgtk-extras
 
    should get you a reasonable OCaml environment to compile Coq.
 
+   Advanced users may want to experiment with the OCaml Flambda
+   compiler as way to improve the performance of Coq, see the entry
+   -flambda-opts below for more details.
```
Comment (Zimmi48) [inline INSTALL]: My comment at https://github.com/coq/coq/pull/7150#discussion_r178812397 applies better to the present PR actually. Would you mind enhancing this paragraph for instance by giving the command to switch to a Flambda compiler using OPAM ?

**Review by Zimmi48 — APPROVED** [review]
I'm OK with merging this PR, but you'll need to rebase #7150 then.

File: INSTALL (hunk 2 above)
Comment (ejgallego) [reply INSTALL]: Done, thanks. No problem in rebasing the other PR.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**